plugins/polio/models.py

```diff
@@ -174,12 +176,31 @@ class RoundDateHistoryEntry(models.Model):
     previous_ended_at = models.DateField(null=True, blank=True)
     started_at = models.DateField(null=True, blank=True)
     ended_at = models.DateField(null=True, blank=True)
+    # Deprecated.
```

Comment:
Couldn't we delete the field then?

Reply:
only after we're sure the powerBi dashboards have been updated accordingly